cmd/agentctl2/utils/print_log.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"html/template"
+	"os"
+
+	"github.com/ligato/vpp-agent/cmd/agentctl/utils"
+)
+
+type logType struct {
+	Logger string `json:"logger,omitempty"`
+	Level  string `json:"level,omitempty"`
+}
+
+type LogList []logType
+
+func ConvertToLogList(log string) LogList {
+	data := make(LogList, 0)
+	err := json.Unmarshal([]byte(log), &data)
+
+	if nil != err {
```

Comment:
Let's start using consistent style here, preferably the common way `err != nil`. Reads weird when reversed.